contracts/components/staking/FortaStaking.sol

```diff
@@ -697,10 +747,17 @@ contract FortaStaking is BaseComponentUpgradeable, ERC1155SupplyUpgradeable, Sub
     }
 
     // Admin: change staking parameters manager
-    function setStakingParametersManager(IStakeController newStakingParameters) public onlyRole(DEFAULT_ADMIN_ROLE) {
-        if (address(newStakingParameters) == address(0)) revert ZeroAddress("newStakingParameters");
-        emit StakeParamsManagerSet(address(newStakingParameters));
-        _stakingParameters = newStakingParameters;
+    function configureStakingHelpers(IStakeSubjectHandler __subjectHandler, IStakeAllocator __allocator) public onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Function name is "configure **staking** helpers" and event name is "**stake** helpers configured".

Let's settle on Stake.

Reply:
bea9f56